src/stepper.js

```diff
@@ -1,6 +1,12 @@
+/* @flow */
 const errorMargin = 0.0001;
 
-export default function stepper(frameRate, x, v, destX, k, b) {
+export default function stepper(frameRate: number,
+                                x: number,
+                                v: number,
+                                destX: number,
+                                k: number,
+                                b: number): Array<number> {
```

Comment:
Are you worried about index out of bounds errors. We can switch the type to `[number, number]`. The docs say tuples are mostly useful for heterogeneous elements, but maybe Flow also checks index validity.